tests/general/KeywordBearTest.py

```diff
@@ -14,3 +22,96 @@
     invalid_files=("test line todo",
                    "test line warNING"),
     settings={"keywords": "todo, warning"})
+
+
+class KeywordBearTodoTest(unittest.TestCase):
+
+    def setUp(self):
+        self.section1 = Section("")
```

Comment:
can we rename that to section? I know we use section1, 2, ... in other tests but that's actually bad style becuase the names don't convey meaning so if you remember a place where we did that you could help us a lot by filing an issue that our code is bad there ;)